Mathlib/Tactic/LibrarySearch.lean

```diff
@@ -49,13 +49,21 @@ initialize librarySearchLemmas : DeclCache (DiscrTree Name true) ←
       let keys ← withReducible <| DiscrTree.mkPath type
       pure $ lemmas.insertCore keys name
 
-/-- Shortcut for calling `solveByElimImpl`. -/
-def solveByElim (g : MVarId) (depth) := SolveByElim.solveByElimImpl false [] depth g
+/-- Shortcut for calling `solveByElim`. -/
+def solveByElim (goals : List MVarId) (required : List Expr) (depth) := do
+  -- I only found a marginal decrease in performance for using the `symm` and `exfalso`
+  -- options for `solveByElim`.
+  -- (measured via `lake build && time lake env lean test/librarySearch.lean`).
+  -- We could nevertheless disable them for only a slight decrease in power.
```

Comment:
I certainly expect `library_search` to try `symm`. I would be against hardcoding `symm := false`, though maybe it would make sense to expose a config option.